synfig-core/po/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+include(${CMAKE_SOURCE_DIR}/cmake/gettext.cmake)
+file(STRINGS LINGUAS PO_LANGUAGES)
+
+gettext_process_po_files(
```

Comment:
I suggest adding a little description of why we do not use the CMake `FindGettext` module for this.

Reply:
I put the description in `/cmake/gettext.cmake`.